fc/protocol.py

```diff
@@ -205,16 +205,16 @@ def _parse(self):
 
         import fc.parsing.CompactSyntaxParser as CSP
         parser = self.parser = CSP.CompactSyntaxParser()
-        CSP.Actions.source_file = self.proto_file
         generator = self.parsed_protocol = parser.try_parse(
             CSP.CompactSyntaxParser.protocol.parseFile,
             self.proto_file,
             parseAll=True
         )[0]
-        assert isinstance(generator, CSP.Actions.Protocol)
-
-        # A class:`fc.parsing.CompactSyntaxParser.Actions.Protocol` object,
+        # A class:`fc.parsing.actions.Protocol` object,
```

Comment:
Missing a `:` in front of "class", but doesn't really need RST as it's not a docstring anyway